src/Serializer/SerializerContextBuilder.php

```diff
@@ -89,6 +91,14 @@ public function createFromRequest(Request $request, bool $normalization, array $
 
         unset($context[DocumentationNormalizer::SWAGGER_DEFINITION_NAME]);
 
+        if (
```

Comment:
does this mean it will not be possible to null a value during a PATCH?
Is this standard PATCH semantics?

Reply:
from my reading, this goes against the json merge patch spec?

Reply:
Actually, it’s the only way to comply with the JSON Merge Patch RFC:

> Null values in the merge patch are given special meaning to indicate the removal of existing values in the target.

>  This design means that merge patch documents are suitable for describing modifications to JSON documents that primarily use objects for their structure and do not make use of explicit null values.  The merge patch format is not appropriate for all JSON syntaxes.

So we need to ignore null values when serializing. Sending patches contains NULL will have the effect to delete the key from the JSON representation of this resource.

Reply:
yep, my bad. i thought this was for deserialization.